CairoMakie/src/primitives.jl

```diff
@@ -932,7 +932,7 @@ function draw_mesh3D(
     projectionview = Makie.space_to_clip(scene.camera, space, true)
     eyeposition = scene.camera.eyeposition[]
     i = Vec(1, 2, 3)
-    normalmatrix = transpose(inv(model[i, i]))
+    normalmatrix = transpose(inv(model[i, i] * Diagonal(Vec3f(scale)))) # see issue #3702
```

Comment:
This is also missing rotation, isn't it? I think we could just replace the diagonal matrix with the local_model matrix from below to fix that as well

Reply:
Please see if it is fixed [here](https://github.com/MakieOrg/Makie.jl/pull/3722/commits/bdaddbdc11a3078333737e47dd74bc3f1eb4f0af).